buildWinDist.bat

```diff
@@ -0,0 +1,8 @@
+rem Make directories
```

Comment:
Move these build scripts into the `scripts` directory, please. I'd like to cleanup the repo root.